### PR TITLE
ISSD-1690 [Marketplace] Show button when CLOUDAPP

### PR DESCRIPTION
**Pull request by eliasantosf**
Task: [ISSD-1690](https://liferay.atlassian.net/browse/ISSD-1690)

### CODE REVIEW

**Comment by eliasantosf** [comment]
ci:test:sf

**Comment by eliasantosf** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/components/PurchasedAppsTable.tsx [hunk 1]
```diff
@@ -183,9 +183,14 @@ const AppsTable: React.FC<AppsTableProps> = ({items}) => {
...
+											disabled={
+												orderTypeExternalReferenceCode !==
+												OrderType.CLOUD
+											}
 											onClick={() => {
-												window.location.href =
-													'https://console.marketplacedemo.liferay.sh/projects';
+												window.open(
+													'https://console.liferay.cloud'
+												);
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/components/PurchasedAppsTable.tsx:193]: We need to leverage CX Properties for that, not having this data hard-coded

Something like this: https://github.com/liferay/liferay-portal/blob/aa4e661cf8d6e38b797edb6617bdba46e2e075e7/modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/main.tsx#L22

https://github.com/liferay/liferay-portal/blob/aa4e661cf8d6e38b797edb6617bdba46e2e075e7/modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/src/main/resources/site-initializer/layouts/testray/page-definition.json#L49

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/components/PurchasedAppsTable.tsx [hunk 2]
```diff
@@ -183,9 +183,14 @@ const AppsTable: React.FC<AppsTableProps> = ({items}) => {
 										</DropDown.Item>
 
 										<DropDown.Item
+											disabled={
+												orderTypeExternalReferenceCode !==
+												OrderType.CLOUD
+											}
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/components/PurchasedAppsTable.tsx:189]: What's the connection between

**Show button when CLOUDAPP** when you are disabling the button?

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by eliasantosf** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/components/PurchasedAppsTable.tsx (hunk 1 above)
Comment (eliasantosf) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/components/PurchasedAppsTable.tsx]: Done!

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/components/PurchasedAppsTable.tsx (hunk 2 above)
Comment (eliasantosf) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/components/PurchasedAppsTable.tsx]: Done!

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**